core/templates/pages/skill-editor-page/questions-tab/skill-questions-tab.directive.ts

```diff
@@ -75,9 +75,23 @@ angular.module('oppia').directive('questionsTab', [
           };
           ctrl.$onInit = function() {
             _init();
-            $scope.$on(EVENT_SKILL_INITIALIZED, _init);
-            $scope.$on(EVENT_SKILL_REINITIALIZED, _init);
+            ctrl.parentSubscription.add(
```

Comment:
```suggestion
            ctrl.parentSubscription.add(
              SkillEditorStateService.getSkillReinitializedSubject().subscribe(
                () => _init()));
```

Reply:
Done!

Reply:
And also, I kept the last `);` in a separate line. to shows where the `ctrl.parentSubscription.add(` is ending.